system/lib/emmalloc.c

```diff
@@ -486,6 +486,12 @@ static bool claim_more_memory(size_t numBytes)
   validate_memory_regions();
 #endif
 
+  // Make sure we send sbrk requests of aligned sizes. If we do not do that then
+  // it will not return contiguous regions, which leads to use creating more
+  // root regions below, inefficiently. (Note that we assume our alignment is
+  // identical to sbrk's, see the assumptions at the start of this file.)
+  numBytes = (size_t)ALIGN_UP(numBytes, MALLOC_ALIGNMENT);
```

Comment:
Is the cast needed here?  numBytes is already size_t.. and I imagine that MALLOC_ALIGNMENT is too, no?